op-challenger/cmd/main.go

```diff
@@ -65,6 +65,24 @@ func run(args []string, action ConfigAction) error {
 		}
 		return action(logger, cfg)
 	}
+	app.Commands = []*cli.Command{
+		{
+			Name: "act",
+			Action: func(ctx *cli.Context) error {
+				logger, err := setupLogging(ctx)
+				if err != nil {
+					return err
+				}
+				logger.Info("Starting op-challenger", "version", VersionWithMeta)
+
+				cfg, err := config.NewConfigFromCLI(ctx)
+				if err != nil {
+					return err
+				}
+				return op_challenger.MainAct(logger, cfg)
+			},
+		},
+	}
```

Comment:
ah, I see why there's a second main entry point now...

I wonder if it would be simpler to just have a `--one-shot` or `--single` CLI option that then goes into the config and the main method switches between calling Loop or Act.  Would save duplicating the init code and isn't an uncommon pattern.

Reply:
yea, this was the thing that was slightly weird. There's also the fact that `op_chalenger.MainLoop` is passed to `run`, but `op_challenger.MainAct` is directly called here.

My preference is to having different sub-commands. I think it's more likely that we want the ability to run an instance of the challenger on a specific game or only have it watch the output. By default it might bundle everything, but I do want to be able to unbundle it.